xgi/generators/uniform.py

```diff
@@ -286,10 +341,26 @@ def uniform_erdos_renyi_hypergraph(n, m, p, p_type="degree", seed=None):
     Hypergraph
         The Erdos Renyi hypergraph
 
-
     See Also
     --------
     ~xgi.generators.random.random_hypergraph
+
+    Notes
+    -----
+    Because XGI only stores edges as sets, if self-loops are allowed,
```

Comment:
But there's no option to allow (or not) self-loops is this function, right? I'm a bit confused. In any case, since that singleton {0} is always removed by the method, does it matter?

The multiedges option is not linked to that correct?